3rdparty/opus-build/opus-build.pro

```diff
@@ -45,19 +45,6 @@ win32 {
 }
 
 unix {
-  CONFIG += staticlib
-  CONFIG(sbcelt) {
-    # Before Opus 1.1 we used to be able to build Opus
-    # as C++ code to get C++ name mangling for free,
-    # allowing us to statically build both libcelt
-    # and libopus into the same binary while avoiding
-    # symbol clashes between the two libraries.
-    #
-    # Stock Opus 1.1 doesn't build in C++ mode, so error
-    # out for now.
-    error(Mumble cannot be built in SBCELT mode with Opus 1.1 - aborting build.)
-  }
```

Comment:
I don't get the logic of this block. Is sbcelt from mumbles qmake build parameters? It bleeds into here?

We always statically link celt 0.7 which doesn't have conflicts? Is this about celt 0.11? Is that yet another lib?

Reply:
> Is sbcelt from mumbles qmake build parameters? It bleeds into here?

Yes.

> We always statically link celt 0.7 which doesn't have conflicts? Is this about celt 0.11? Is that yet another lib?

We dynamically link to both CELT 0.7 and CELT 0.11.

The `sbcelt` build option enables a special version of CELT, statically linked.

https://github.com/mumble-voip/sbcelt